### ISSUE TITLE
How to download figures from plotly.offline.iplot after setting atttribute image='svg'

### ISSUE THREAD

**Issue by sw-gong** [plotly/plotly.py#1502]
It is said from the `help(plotly.offline.iplot)`:
>    image (default=None |'png' |'jpeg' |'svg' |'webp') -- This parameter sets
        the format of the image to be downloaded, if we choose to download an
        image. This parameter has a default value of None indicating that no
        image should be downloaded.

My understanding is that we can download the figure as the format of the image after setting this attribute. However, how can we download it? Please see the below code. Could anyone explain what should be down next in order to download the figure?

Alternatively, I can use `plotly.offline.plot` which return a _html_ link and I can download the figure after openning the link. However, the downside is that the figure won't be shown inside the jupyter notebook. 

```
import plotly
import plotly.graph_objs as go
init_notebook_mode(True)

img_name = 'my-plot'
data = [go.Scatter(x=[1, 2, 3], y=[3, 2, 6])]
plotly.offline.iplot(data, filename=img_name, image='svg')
```

**Comment by jonmmease** [comment]
Hi @sw-gong,

Looks like this was a bug introduced back in https://github.com/plotly/plotly.py/pull/1250.  This will be fixed in version 3.8 by https://github.com/plotly/plotly.py/pull/1474.

That said, our recommended approach for exporting figures as static images is to use `plotly.io.write_image` which relies on the orca utility.  See https://plot.ly/python/static-image-export/ for a full description.

**Comment by jonmmease** [comment]
Fixed in https://github.com/plotly/plotly.py/pull/1474
